sdk/core/AzureCore/Tests/LoggingPolicyTests.swift

```diff
@@ -179,17 +179,6 @@ class LoggingPolicyTests: XCTestCase {
         XCTAssertEqual(msg?.text, "GET http://www.example.com?id=123&test=REDACTED")
     }
 
-    /// Test that the logging policy logs internal failures at the correct level
-    func test_LoggingPolicy_OnRequestWithInvalidUrl_LogsFailureAtWarningLevel() {
```

Comment:
Why was this test removed?

Reply:
The test was failing. After poking at this for nearly an hour, I determined that I couldn't construct any string which would cause the construction of the URL to fail any more, because the `HTTPRequest` initializer now calls `addingPercentEncoding` to the string URL parameter, and try as I might to feed it something that it would choke on, I could never get it to -not- return a URL instance.